libs/core/src/utils/overlay.ts

```diff
@@ -0,0 +1,174 @@
+export const positionTooltip = (elem, elemPlacement, overlay, offset = 8) => {
```

Comment:
I'd love to learn more about utils and how we should be using them.

Reply:
Thanks for the time and convo on this today. Extremely helpful!

Reply:
It started as a function in the Tooltip component. Once I realized that it could be used in another component, I added it to a folder outside the component to then import it. `utils` serves more as the shared directory to reduce the need for duplicated code across components.